src/talos/annotation_scripts/ReformatAnnotatedVcfIntoHailTable.py

```diff
@@ -279,7 +279,7 @@ def main(
         init_batch()
     else:
         logger.info('Using Hail Local backend, no checkpoints')
-        hl.context.init_spark(master='local[2]', default_reference='GRCh38', quiet=True)
+        hl.context.init_spark(master='local[*]', default_reference='GRCh38', quiet=True)
```

Comment:
This opens up the Spark instance to use all available cores, instead of needing to be updated in sync with the VM provision